tests/examples/wibox/widget/calendar/year.lua

```diff
@@ -0,0 +1,24 @@
+local parent    = ... --DOC_HIDE
+local wibox     = require("wibox") --DOC_HIDE
+local beautiful = require( "beautiful" ) --DOC_HIDE
+local lgi = require("lgi") --DOC_HIDE
+local Pango = lgi.Pango --DOC_HIDE
```

Comment:
You can probably consolidate this and the previous line to:
`local Pango = require("lgi").Pango`
since lgi isn't used anywhere except here.